### DISCUSSION TITLE
I want to order by base on specific values provided in In query

### DISCUSSION THREAD

**Discussion by sandbox-apps** [prisma/prisma#8470] [Q&A] [answered]
I want to order by base on specific values provided in In query
Example if I have id 2, 3, 1
I want my result to be ordered by 2, 3, 1 also

**Comment by ryands17** [comment, marked as answer]
@sandbox-apps 👋 

The database by default orders by `id` so this isn't possible. Here's an example using Postgres:

![image](https://user-images.githubusercontent.com/19697099/127276230-244e1697-2faa-4b9a-990d-7f4eb11e0c5a.png)

So you cannot have the same order as what you provided in `in`.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
